README.md

```diff
@@ -23,6 +24,12 @@ Check out our [tutorial on configuring and building these samples on Ubuntu](htt
 	- [**ReadPCLVis3D**](https://github.com/zivid/cpp-extra-samples/blob/master/ZividPCL/ReadPCLVis3D/ReadPCLVis3D.cpp) - This example shows how to read a PCL point cloud and visualize it.
 	- [**CaptureWritePCLVis3D**](https://github.com/zivid/cpp-extra-samples/blob/master/ZividPCL/CaptureWritePCLVis3D/CaptureWritePCLVis3D.cpp) - This example shows how capture a Zivid point cloud, save it to a .PCD file format, and visualize it.
 	- [**CaptureFromFileWritePCLVis3D**](https://github.com/zivid/cpp-extra-samples/blob/master/ZividPCL/CaptureFromFileWritePCLVis3D/CaptureFromFileWritePCLVis3D.cpp) - This example shows how capture a Zivid point cloud from an emulated Zivid camera, save it to a .PCD file format, and visualize it.
+	- **Dependencies:**
+		- [PCL](http://pointclouds.org/) version 1.9.1
+
+- [**Downsample**](https://github.com/zivid/cpp-extra-samples/tree/master/Downsample)
+	- **Dependencies:**
+		- [Eigen](http://eigen.tuxfamily.org/) version 3.3.90
```

Comment:
I guess we don't require exactly this version? Probably 3.4.0 will work also etc. Maybe just say "version 3.3.90 or newer" or something like that. Managing versions of dependencies properly is hard, but I think that should be good enough for a sample like this.

Reply:
Done